frontend/apps/web/app/[account]/transformers/components/UserDefinedTransformersTable/columns.tsx

```diff
@@ -52,7 +52,14 @@ export function getUserDefinedTransformerColumns(
         return (
           <div className="flex space-x-2">
             <span className="max-w-[500px] truncate font-medium">
-              {row.original.name}
+               <div>
```

Comment:
let's just make the name clickable. no need to make the datatype or source clickable.

Reply:
Actually you said in the first comment on the issue to make all the resources clickable too.

Reply:
sorry for the confusion. I was referring to the resource's `name` to be clickable. We used to have the identifier in the column but it was removed to make the table look cleaner.